src/raster/r.viewshed.exposure/r.viewshed.exposure.html

```diff
@@ -0,0 +1,222 @@
+and solid angle (example 2). Sampling density is set to 50%,
+exposure range to 2km.
+
+<div class="code"><pre>
+# set computation region to terrain model
+g.region raster=elevation@PERMANENT
+
+# calculate visual exposure
+# no viewshed parametrisation function (binary viewshed)
+r.viewshed.exposure dsm=elevation@PERMANENT
+  output=exposure_roadsmajor_b
+  source=roadsmajor@PERMANENT
+  observer_elevation=1.50
+  max_distance=2000
+  sample_density=50 memory=5000 cores=25
+
+# calculate visual exposure
+# solid anfle viewshed parametrisation function
+r.viewshed.exposure dsm=elevation@PERMANENT
+  output=exposure_roadsmajor_s
+  source=roadsmajor@PERMANENT
+  observer_elevation=1.50
+  max_distance=2000
+  function=solid_angle
+  sample_density=50 memory=5000 cores=25
+
+# scale solid angle values for visualisation purposes
+# (see Domingo-Santos et al., 2011)
+r.mapcalc expression=exposure_roadsmajor_s_rescaled =
+  if(exposure_roadsmajor_s@user1>=0.2*3.1416,1,1/
+  (-1* log(exposure_roadsmajor_s@user1 /(2*3.1416))))
+
+</pre></div>
+
+<div align="center" style="margin: 10px">
+<a href="r_viewshed_exposure_example_binary.png">
+<img src="r_viewshed_exposure_example_binary.png"
+width="601" height="747"
+alt="Example of r.viewshed.exposure (1)" border="0">
+</a><br>
+<i>Example of r.viewshed.exposure (1)</i>
+</div>
+
+<div align="center" style="margin: 10px">
+<a href="r_viewshed_exposure_example_solid_angle.png">
+<img src="r_viewshed_exposure_example_solid_angle.png"
+width="601" height="747"
+alt="Example of r.viewshed.exposure (2)" border="0">
+</a><br>
+<i>Example of r.viewshed.exposure (2)</i>
+</div>
+
+<h2>TODO</h2>
+<ul>
+  <li>Implement variable exposure source height.</li>
+  <li>Implement possibility to switch between absolute and relative values
+    of visual exposure (now absolute).</li>
+</ul>
+<h2>KNOWN ISSUES</h2>
+
+<h2>REFERENCES</h2>
+
+<ul>
+<li>Cimburova, Z., Blumentrath, S., 2022.
+  Viewshed-based modelling of visual exposure to urban greenery -
+  an efficient GIS tool for practical appliactions (in press).
+  Landscape and Urban Planning.
+</li>
+<li>Chamberlain, B.C., Meitner, M.J., 2013.
+  A route-based visibility analysis for landscape management.
+  <i>Landscape and Urban Planning</i> 111, 13-24.
+  <a href="https://doi.org/10.1016/j.landurbplan.2012.12.004">
+    https://doi.org/10.1016/j.landurbplan.2012.12.004 </a> </li>
+<li>Domingo-Santos, J.M., de Villarán, R.F., Rapp-Arrarás, Í., de Provens,
+  E.C.-P., 2011.
+  The visual exposure in forest and rural landscapes:
+  An algorithm and a GIS tool.
+  <i>Landscape and Urban Planning</i> 101, 52-58.
+  <a href="https://doi.org/10.1016/j.landurbplan.2010.11.018">
+  https://doi.org/10.1016/j.landurbplan.2010.11.018</a></li>
+<li>Fisher, P., 1994.
+  Probable and fuzzy models of the viewshed operation,
+  in: Worboys, M.F. (Ed.), <i>Innovations in GIS</i>.
+  Taylor & Francis, London, pp. 161-176.</li>
+<li>Grêt-Regamey, A., Bishop, I.D., Bebi, P., 2007.
```

Comment:
```suggestion
<li>Grêt-Regamey, A., Bishop, I.D., Bebi, P., 2007.
```

Please use HTML code for ê etc.

Reply:
Thanks. Should be solved now.